tests/src/test/scala/com/nvidia/spark/rapids/HashAggregatesSuite.scala

```diff
@@ -35,7 +36,12 @@ class HashAggregatesSuite extends SparkQueryCompareTestSuite {
   }
 
   private def checkExecNode(result: DataFrame): Unit = {
-    if (result.queryExecution.executedPlan.conf.getAllConfs(RapidsConf.SQL_ENABLED.key).toBoolean) {
+    if (result.queryExecution.executedPlan.conf.getAllConfs(RapidsConf.SQL_ENABLED.key).toBoolean
+      && !SparkSessionHolder.adaptiveQueryEnabled) {
+
+      // this assertion is skipped when AQE is enabled because it is looking at the plan before
```

Comment:
is there a later time where the HashAgg would become a GpuHashAgg after AQE?

Reply:
Yes. With AQE enabled, the plan transformation to GPU just happens later, at execution time rather than at initial planning time. I'm actually refactoring these tests now to look at the plan after execution instead (which works with or without AQE enabled).

Reply:
>  I'm actually refactoring these tests now to look at the plan after execution instead

Is this a moving target? Or do you get snapshots of different executed plans?

Reply:
Resolving this one, I had a comment in the code later.